persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraKeyspaceConfig.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.event.LoggingAdapter
+import com.typesafe.config.Config
+
+private[lagom] object CassandraKeyspaceConfig {
+
+  def validateKeyspace(namespace: String, defaultNamespace: String, config: Config, log: LoggingAdapter): Unit = {
+    if (log.isWarningEnabled) {
+      val keyspace = config.getString(s"$namespace.keyspace")
+      val defaultPath = s"$defaultNamespace.keyspace"
+      if (config.hasPath(defaultPath) && keyspace == config.getString(defaultPath)) {
+        log.warning(
```

Comment:
After sleeping on it, I changed my mind and decided that verbosity is OK here and it's important to be clear. What do you think?